app/controllers/application_controller.rb

```diff
@@ -167,7 +167,7 @@ def controller_permission
 
   def action_permission
     case params[:action]
-      when 'new', 'create'
+      when 'new', 'create', 'clone'
```

Comment:
This should be in the media_controller, not here.
Same as https://github.com/theforeman/foreman/pull/9397/files#r963420270